README.md

```diff
@@ -1,13 +1,11 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Here is my first Backend project, I have made three endpoints and a front end is to be added :)
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The diffrence between using patch nad put endpoints was confusing in the beginning.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://nazanin-project-express-api.herokuapp.com/
```

Comment:
It would be very nice to have some instructions, how to use your API. Remember to make documentation on what endpoints and queries your user can use. 🤓